bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/checkboxx/CheckBoxX.java

```diff
@@ -128,6 +130,7 @@ public CheckBoxXConfig getConfig() {
     public void renderHead(IHeaderResponse response) {
         super.renderHead(response);
 
+        response.render(JavaScriptHeaderItem.forReference(Application.get().getJavaScriptLibrarySettings().getJQueryReference()));
```

Comment:
This change could be simplified to:
response.render(JavaScriptHeaderItem.forReference(new `JQueryPluginResourceReference`(CheckBoxX.class, "js/checkbox-x.js")));